recipe-server/client/control/utils/apiFetch.js

```diff
@@ -5,7 +5,9 @@ export default async function apiFetch(url, options = {}) {
 
   const headers = new Headers();
   headers.append('Accept', 'application/json');
-  headers.append('Content-Type', 'application/json');
+  if (!(options.body && options.body instanceof FormData)) {
```

Comment:
yeah i just wanted to get this working but I think a refactor of this method makes sense.